docs/node-operators/requirements/data-node-infrastructure.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 2
+title: Data node infrastructure
+sidebar_label: Data node infrastructure
+vega_network: TESTNET
+hide_title: false
+---
+import NetworkParameter from '@site/src/components/NetworkParameter';
+import Topic from '/docs/topics/_topic-data-nodes.mdx';
+
+<Topic />
+
+This page covers the minimum and recommended hardware requirements, and the software requirements for running a data node connecting to the Vega network.
+
+The best hardware may vary depending on how you set up your node and database service, and how you intend to use the data node.
+
+Data node operators who are running a data node to support their own needs can run a leaner setup depending on their own requirements.
+
+Those data node operators running archive nodes, which retain all chain information, will need a more robust setup.
+
+## Hardware recommendations
+
+### System requirements
+| Resource    | Minimum     | Recommended |
+| ----------- | ----------- | ----------- |
+| CPU | 6 cores at >= 3Ghz | 12 cores at >= 4Ghz|
+| RAM   | 16GB            | 32GB        |
+| Storage   | 1TB NVMe SSD| 2TB NVMe SSD |
+
+The impact of using fewer cores than recommended is that the critical parts (data node and non-validators) will be starved of CPU resources, which will reduce the maximum throughput of the network.
+
+The impact of having slower cores than recommended (or older cores that have a reduced IPC rate compared to modern Zen3 cores) is that the maximum throughput will be reduced due to the bottlenecks being single threaded.
+
+### File system performance testing
+In benchmark testing using the default Ubuntu file system of EXT4 against ZFS with zstd compression turned on, it was determined that ZFS has an impact on performance when using NVMe drives, while the opposite is seen in SATA drives.
```

Comment:
**thought (non-blocking)**: I'm not all that familiar with zstd so my initial thought was that linking off to somewhere (e.g. a github page or some zfs docs for enabling this) might be good. But I think that's a step too far in to hand holding.

Reply:
There will be people out there who do not know what EXT4 is as well, but I think if they are considering changing their filesystem from the default they need to go and and do some reading by themselves as they will need to understand how to create new partitions and format them correctly. This would be too complicated for this doc.